frontend-react/src/App.tsx

```diff
@@ -14,7 +14,7 @@ import { Route, useHistory, Switch } from "react-router-dom";
 import { OktaAuth, toRelativeUrl } from "@okta/okta-auth-js";
 import { Security, SecureRoute, LoginCallback } from "@okta/okta-react";
 import { NetworkErrorBoundary } from "rest-hooks";
-import { SpinnerCircular } from "spinners-react";
+import Spinner from './components/Spinner'
```

Comment:
Seems to be unused in this file?

Also the `import { Suspense } from "react";` from line 11 is now unused?